packages/atomic/cypress/integration/facets-v1/facet/facet.cypress.ts

```diff
@@ -107,6 +108,124 @@ describe('Facet v1 Test Suites', () => {
           });
         });
       });
+
+      describe('when searching for a value that returns results', () => {
+        const query = 'bbc';
+        function setupSearchFor() {
+          setupSelectCheckboxValue();
+          typeFacetSearchQuery(query);
+        }
+
+        describe('verify rendering', () => {
+          before(setupSearchFor);
+
+          FacetAssertions.assertAccessibility();
+          FacetAssertions.assertNumberOfIdleCheckboxValues(
+            defaultNumberOfValues
+          );
+          FacetAssertions.assertNumberOfSelectedCheckboxValues(0);
+          FacetAssertions.assertDisplayMoreMatchesFound(true);
+          FacetAssertions.assertDisplayNoMatchesFound(false);
+          FacetAssertions.assertMoreMatchesFoundContainsQuery(query);
+          FacetAssertions.assertDisplayShowMoreButton(false);
+          FacetAssertions.assertDisplaySearchClearButton(true);
+        });
+
+        describe('verify analytics', () => {
+          before(setupSearchFor);
+
+          FacetAssertions.assertLogFacetSearch(field);
+        });
+
+        describe('when selecting for a search result', () => {
+          function setupSelectSearcResult() {
+            setupSearchFor();
+            cy.wait(TestFixture.interceptAliases.UA);
+            selectIdleCheckboxValueAt(5);
+            cy.wait(TestFixture.interceptAliases.Search);
+          }
+
+          describe('verify rendering', () => {
+            before(setupSelectSearcResult);
+
+            FacetAssertions.assertNumberOfIdleCheckboxValues(
+              defaultNumberOfValues - 2
+            );
+            FacetAssertions.assertNumberOfSelectedCheckboxValues(2);
+            FacetAssertions.assertDisplayMoreMatchesFound(false);
+            FacetAssertions.assertDisplayNoMatchesFound(false);
+            FacetAssertions.assertDisplayShowMoreButton(true);
```

Comment:
If no more matches are found, should the `ShowMoreButton` be displayed?

Reply:
There is never a show more button when searching, rather its the "More Matches"/"No Matches" label or nothing.
Here once you select a search results, the search clears, and the normal values are showed.

![Screen Shot 2021-06-16 at 9 26 52 AM](https://user-images.githubusercontent.com/4923043/122227574-fec10680-ce84-11eb-9a7e-fcaad342d731.png)